mkdocs.yml

```diff
@@ -75,6 +76,7 @@ nav:
               - Baseline Parameters: spec/0.95.0/4_Baseline_Parameters.md
               - Ballot Encryption: spec/0.95.0/6_Ballot_Encryption.md
               - Ballot Aggregation: spec/0.95.0/7_Ballot_Aggregation.md
+              - Verifiable Decryption: spec/0.95.0/8_Verifiable_Decryption.md
```

Comment:
@anirudhprabhakaran3 
We made some slight adjustments here. 

- [ ] Move the file to `spec/web`
- [ ] Shift the mkdocs to `spec/web/8_Verifiable_Decryption.md` . There should only be one reference now.